src/__tests__/standalone-syntax-test.js

```diff
@@ -133,3 +133,87 @@ test("standalone with syntax set by extension", t => {
 
   t.end()
 })
+
+test("standalone with path to custom parser", t => {
+  const config = {
+    rules: {
+      "block-no-empty": true,
+    },
+  }
+
+  standalone({
+    config,
+    syntax: `${fixturesPath}/custom-parser`,
+    code: ".foo { width: 200px; }\n.bar {}",
+    formatter: stringFormatter,
+  }).then(({ output }) => {
+    const strippedOutput = chalk.stripColor(output)
+    t.equal(typeof output, "string")
+    t.ok(strippedOutput.indexOf("2:6") !== -1)
+    t.ok(strippedOutput.indexOf("block-no-empty") !== -1)
```

Comment:
> we could use something like the postcss-safe-parser and have it try to parse something that the regular postcss parser can't.

Yeah, that's what I think we should do. Set up a situation so that there will be a parsing problem if the right parser is not used; or the linting output will change. Because we're not just testing whether the syntax module is *found*, but whether it is used to change the parsing and the eventual output.